Documentation/kube-flannel.yml

```diff
@@ -149,6 +149,7 @@ spec:
         tier: node
         app: flannel
     spec:
+      priorityClassName: system-cluster-critical
```

Comment:
according to `system-cluster-critical` description , use `system-node-critical` maybe better

```
apiVersion: scheduling.k8s.io/v1
description: Used for system critical pods that must run in the cluster, but can be
  moved to another node if necessary.
kind: PriorityClass
metadata:
  creationTimestamp: "2019-01-13T08:16:42Z"
  generation: 1
  name: system-cluster-critical
  resourceVersion: "19528377"
  selfLink: /apis/scheduling.k8s.io/v1/priorityclasses/system-cluster-critical
  uid: 8f120be6-170b-11e9-b496-38d54701236f
value: 2000000000

---

apiVersion: scheduling.k8s.io/v1
description: Used for system critical pods that must not be moved from their current
  node.
kind: PriorityClass
metadata:
  creationTimestamp: "2019-01-13T08:16:42Z"
  generation: 1
  name: system-node-critical
  resourceVersion: "19528376"
  selfLink: /apis/scheduling.k8s.io/v1/priorityclasses/system-node-critical
  uid: 8f11339f-170b-11e9-b496-38d54701236f
value: 2000001000
```

Reply:
Seems you are right on that. Can this be merged if I change that line?

Reply:
changed